src/draw.jl

```diff
@@ -1,6 +1,6 @@
 ## draw: unify rand, sample
 
-draw!(x::T, o) where {T<:Distribution} = rand!(x, o)
```

Comment:
Is this related to performance or aesthetics?

Reply:
There's no performance difference but it's [recommended to remove unnecessary static parameters](https://docs.julialang.org/en/v1/manual/style-guide/#Don't-use-unnecessary-static-parameters-1).